example/ruby/sample.rb

```diff
@@ -0,0 +1,5 @@
+require 'openssl'
+require './github'
+
```

Comment:
You're right. I thought that it is basic implementation app using external APIs. But embedding api client into sample.rb is better.
I'll fix it.